src/languages/cpp.js

```diff
@@ -218,11 +218,31 @@ export default function(hljs) {
     'stderr',
     'stringstream',
     'istringstream',
-    'ostringstream'
+    'ostringstream',
+    'optional',
+    'pair',
+    'tuple',
+    'make_tuple',
+    'any',
+    'function',
+    'variant',
+    'visit',
+    'apply',
+    'as_const',
+    'to_underlying',
+    'swap',
+    'exchange',
+    'forward',
+    'move',
+    'declval'
   ];
 
   const CPP_KEYWORDS = {
     keyword: 'int float while private char char8_t char16_t char32_t catch import module export virtual operator sizeof ' +
+      'int8_t int16_t int32_t int64_t int_fast8_t int_fast16_t int_fast32_t int_fast64_t ' +
```

Comment:
These should already be handled by `CPP_PRIMITIVE_TYPES`, no?

Reply:
Yes, you're right, forgive me. I will update the list.

I'm wondering though why the `charXX_t` (and other `_t` types) are there in the list as well then. They should all be handled by `CPP_PRIMITIVE_TYPES`. Perhaps they can be removed.

Reply:
> I'm wondering though why the charXX_t (and other _t types) are there in the list as well then. 

Me too, I'd say they could safely be removed?

Reply:
Yes, I'll check this and extend the test for this. There currently is a test that only tries a single example.

Reply:
This should be solved in the latest version now.